app/grandchallenge/archives/models.py

```diff
@@ -359,6 +359,13 @@ def delete_url(self):
             kwargs={"slug": self.base_object.slug, "pk": self.pk},
         )
 
+    @property
```

Comment:
`get_absolute_url` should not be a property. https://docs.djangoproject.com/en/5.0/ref/models/instances/#get-absolute-url